src/cache.h

```diff
@@ -92,6 +92,16 @@ extern char *META_DIR;
  */
 void CacheSystem_init(const char *path, int url_supplied);
 
+/**
+ * \brief clear the cache system directory
+ * \details This function basically clears these paths on disk:
+ *  - META_DIR
+ *  - DATA_DIR
+ *
+ * If these directories exist, they will be deleted.
```

Comment:
I want all the content of the cache directory removed, not the META_DIR and DATA_DIR of a website.

Reply:
Comment updated. The entire cache directory is in fact removed.